tests_e2e/pipeline/pipeline.yml

```diff
@@ -10,9 +37,50 @@ pr: none
 pool:
   vmImage: ubuntu-latest
 
-stages:
-  - stage: "ExecuteTests"
+jobs:
```

Comment:
The prototype pipeline was significantly larger than the new pipeline and was using stages and templates to organize the steps. This is not needed in the new pipeline so I am removing the stages and coalescing the templates